README.md

```diff
@@ -12,6 +12,15 @@ Unlike traditional operating systems, DC/OS spans multiple machines within a net
 To learn more, see the [DC/OS Overview](https://docs.d2iq.com/mesosphere/dcos/latest/overview/).
 
 
+# Project status
+
+D2iQ (formerly Mesosphere), that created and sponsored this project from the very beginning, has [announced on October 30th, 2020](https://d2iq.com/blog/d2iq-takes-the-next-step-forward):
+
+> Starting today, we will begin the process to wind down the Mesosphere DC/OS Platform. We will continue to provide support for our current DC/OS customers, of course, but will no longer be investing in new features or capabilities. We are working closely with existing customers to migrate […]”
+
+**Future of this project is for now uncertain**, and it might be a good idea to consider the above announcement when evaluating if DC/OS is the right solution for future deployments.
```

Comment:
```suggestion
D2iQ (formerly Mesosphere) created and sponsored this project from the very beginning, but we [announced on October 30th, 2020](https://d2iq.com/blog/d2iq-takes-the-next-step-forward):

> As of <date>, we are winding down the Mesosphere DC/OS Platform. We will continue to provide support for our current DC/OS customers, but will no longer be investing in new features or capabilities. We continue to work closely with existing customers to migrate […]”

**Future of this project is for now uncertain**, and it might be a good idea to consider the above announcement when evaluating if DC/OS is the right solution for future deployments.
```

Reply:
Perhaps we should replace "today" with the blog's date?

> As of [Oct 30, 2020], we ...